### ISSUE TITLE
buttons suck

### ISSUE THREAD

**Issue by CamsShaft** [ruihq/SmartQ#6]
This would be great except the login and register buttons don't work :( can I decompile with apktool m and fix and how do I do that? Cheers!

**Comment by ruingl** [comment]
yeah u can decomp, this is an old project tho so it wont be getting updates, but thx for the issue

**Comment by CamsShaft** [comment]
Thanks for the quick reply. I realized it was "old" by today's standards after lol. I found an openai gpt on apkmirror but it's the same as all the other ones I've tried. It keeps giving me an error saying something about the date and time being wrong and then wont log in which is just dumb. Have you experienced any of that before? Could it be because I got it from a third party site?

**Comment by ruingl** [comment]
yes. i have experienced it. oh, try to update your date and time on your settings, if ur using android theres a automate time there i think

**Comment by ruingl** [comment]
also, i do think this app doesnt work cause it doesnt have cookies enabled in the code

**Comment by ruingl** [comment]
and it uses an old webview from android, this shi doesnt even use androidx lmaoo

**Comment by CamsShaft** [comment]
Yeah I've got three settings on my phone all switched to automatic. I've messed around with it not too long ago and nothing helped. Haha I think your repo is dead now that you said that. It's weird because on gpt's website I can login no problem. I'm just having a hard time figuring it out. I just want to make writing scripts easier instead of taking "hours" and still fu**ing up lol

**Comment by ruingl** [comment]
you could've just used the appstore one ngl

**Comment by CamsShaft** [comment]
I just can't stand google apps and everytime I try to use apktool to try to edit apps they end up installing but then closing right away. Plus there's a ton of trackers in their apps. I'm just using the web gpt which is alright for now. Maybe I'll just ask it to help me fix itself haha

**Comment by ruingl** [comment]
they prob use proguard to stop deobfuscators
